src/main/java/nz/ac/waikato/its/dspace/app/xmlui/aspect/featuredcontent/CocoonFlowAspectDebug.java

```diff
@@ -0,0 +1,22 @@
+package nz.ac.waikato.its.dspace.app.xmlui.aspect.featuredcontent;
```

Comment:
Would you mind moving this class to the package `nz.ac.waikato.its.dspace.app.xmlui.util` please? That way we can use it from other aspects without getting confusing imports.